docs/deployments-aws.md

```diff
@@ -0,0 +1,168 @@
+# Deploying services to AWS
+
+The code to deploy our [Live site](https://cudl.lib.cam.ac.uk) is stored in GitHub at https://github.com/cambridge-collection/cudl-terraform. 
+
+## Components
+
+The Terraform code deploys the following as a series of Terraform modules:
+
+- Base architecture, including an AWS Virtual Private Cloud (VPC), Web Application Firewall (WAF), Autoscaling Group (ASG), Application Load Balancer (ALB) and an Elastic Container Service (ECS) cluster.
+- Data processing, including S3 Buckets, Lambda Functions and Simple Queue Service (SQS) queues to deliver data for use by the Viewer.
+- ECS workloads, including task definitions, Amazon Certificate Manager (ACM) certificates, Route 53 records (for DNS), CloudFront Distribution (handling CDN), ALB listeners and target groups, Identity Access Management (IAM) permissions for tasks and Elastic File System (EFS) volumes.
+
+## Terraform Modules
+
+Three Terraform modules are used in the deployment:
+
+- Base Architecture: https://github.com/cambridge-collection/terraform-aws-architecture-ecs
+- Data Processing: https://github.com/cambridge-collection/cudl-terraform/tree/main/modules/cudl-data-processing
+- ECS Workloads: https://github.com/cambridge-collection/terraform-aws-workload-ecs
+
+## Container Images
+
+The ECS workloads and several lambdas are dependent on container images deployed to the AWS Elastic Container Repository (ECR). Images are managed outside of Terraform. The following container images are used:
+
+- Lambdas:
+    - SOLR listener lambdas: https://github.com/cambridge-collection/cudl-solr-listener
+    - TEI Processing: https://github.com/cambridge-collection/cudl-data-processing-xslt
+    - Transkribus:
```

Comment:
I believe it's:
- Transkribus: https://github.com/cambridge-collection/curious-cures-transkribus-lambda

Reply:
aha that's the one thanks